keras/backend/mxnet_backend.py

```diff
@@ -3628,6 +3710,8 @@ def _get_type(self):
     def __getitem__(self, in_slice):
         begin = []
         end = []
+        if not isinstance(in_slice, (list, tuple)):
```

Comment:
Why?

Reply:
`in_slice` should be a tuple or list of “slice()” constructor. But it fails(Single dimensional bias Tensors) if `in_slice` is not a tuple or list(Iterative object).

Reply:
Please add this description as code comment.

Reply:
Done. Thanks!